--- minizip/unzip.c	2012-10-22 11:55:02 +0000
+++ minizip/unzip.c	2012-10-22 11:55:25 +0000
@@ -1145,7 +1145,7 @@
                                                 szFileName,fileNameBufferSize,
                                                 extraField,extraFieldBufferSize,
                                                 szComment,commentBufferSize);
-    if (err==UNZ_OK)
+    if (err==UNZ_OK && pfile_info)
     {
         pfile_info->version = file_info64.version;
         pfile_info->version_needed = file_info64.version_needed;
