--- include/ode/common.h	Mon Jun  2 23:09:37 2008
+++ include/ode/common.h	Wed Aug  8 08:43:51 2012
@@ -370,6 +370,7 @@
   dVector3 t1;		/* torque applied to body 1 */
   dVector3 f2;		/* force applied to body 2 */
   dVector3 t2;		/* torque applied to body 2 */
+  dReal lambda;
 } dJointFeedback;
 
 
--- ode/src/step.cpp	Thu Jul 24 22:18:40 2008
+++ ode/src/step.cpp	Mon Oct 15 12:58:57 2012
@@ -762,7 +762,8 @@
             cf2[4] += (fb->t2[0] = data[4]);
             cf2[5] += (fb->t2[1] = data[5]);
             cf2[6] += (fb->t2[2] = data[6]);
           }
+          fb->lambda = *(lambdarow+5);
         }
         else {
 	       // no feedback is required, let's compute cforce the faster way
