--- ode/src/error.cpp	2013-02-08 10:21:03.886689559 +0100
+++ ode/src/error.cpp	2013-02-08 10:22:33.438692195 +0100
@@ -90,7 +90,8 @@
   if (error_function) error_function (num,msg,ap);
   else printMessage (num,"ODE Error",msg,ap);
   va_end (ap);
-  exit (1);
+  if (!error_function)
+    exit (1);
 }
 
 
@@ -102,7 +103,8 @@
   else printMessage (num,"ODE INTERNAL ERROR",msg,ap);
   // *((char *)0) = 0;   ... commit SEGVicide
   va_end (ap);
-  abort();
+  if (!debug_function)
+    abort();
 }
 
 
@@ -145,7 +147,8 @@
     MessageBox(0,s,title,MB_OK | MB_ICONWARNING);
   }
   va_end (ap);
-  exit (1);
+  if (!error_function)
+    exit (1);
 }
 
 
@@ -162,7 +165,8 @@
     MessageBox(0,s,title,MB_OK | MB_ICONSTOP);
   }
   va_end (ap);
-  abort();
+  if (!debug_function)
+    abort();
 }
 
 
