--- ode/src/error.cpp	Mon Oct 15 13:20:28 2012
+++ ode/src/error.cpp	Mon Oct 15 13:20:38 2012
@@ -89,6 +89,7 @@
   va_start (ap,msg);
   if (error_function) error_function (num,msg,ap);
   else printMessage (num,"ODE Error",msg,ap);
+  va_end (ap);
   exit (1);
 }
 
@@ -100,6 +101,7 @@
   if (debug_function) debug_function (num,msg,ap);
   else printMessage (num,"ODE INTERNAL ERROR",msg,ap);
   // *((char *)0) = 0;   ... commit SEGVicide
+  va_end (ap);
   abort();
 }
 
@@ -110,6 +112,7 @@
   va_start (ap,msg);
   if (message_function) message_function (num,msg,ap);
   else printMessage (num,"ODE Message",msg,ap);
+  va_end (ap);
 }
 
 #endif
@@ -141,6 +144,7 @@
     s[sizeof(s)-1] = 0;
     MessageBox(0,s,title,MB_OK | MB_ICONWARNING);
   }
+  va_end (ap);
   exit (1);
 }
 
@@ -157,6 +161,7 @@
     s[sizeof(s)-1] = 0;
     MessageBox(0,s,title,MB_OK | MB_ICONSTOP);
   }
+  va_end (ap);
   abort();
 }
 
@@ -167,6 +172,7 @@
   va_start (ap,msg);
   if (message_function) message_function (num,msg,ap);
   else printMessage (num,"ODE Message",msg,ap);
+  va_end (ap);
 }
 
 
--- ode/demo/demo_ode.cpp	Mon Oct 15 13:17:15 2012
+++ ode/ode/demo/demo_ode.cpp	Mon Oct 15 13:17:26 2012
@@ -951,6 +951,7 @@
     va_list ap;
     va_start (ap,name);
     vsprintf (mi->name,name,ap);
+    va_end (ap);
     if (strlen(mi->name) >= sizeof (mi->name)) dDebug (0,"name too long");
 
     mat.push_back(mi);
@@ -967,6 +968,7 @@
     va_list ap;
     va_start (ap,name);
     vsprintf (mi.name,name,ap);
+    va_end (ap);
     if (strlen(mi.name) >= sizeof (mi.name)) dDebug (0,"name too long");
 
     if (strcmp(mp->name,mi.name) != 0)
--- drawstuff/src/osx.cpp	Mon Oct 15 13:18:38 2012
+++ drawstuff/src/osx.cpp	Mon Oct 15 13:18:48 2012
@@ -79,6 +79,7 @@
   va_list ap;
   va_start (ap,msg);
   printMessage ("Error",msg,ap);
+  va_end (ap);
   exit (1);
 }
 
@@ -89,6 +90,7 @@
   va_start (ap,msg);
   printMessage ("INTERNAL ERROR",msg,ap);
   // *((char *)0) = 0;	 ... commit SEGVicide ?
+  va_end (ap);
   abort();
 }
 
@@ -97,6 +99,7 @@
   va_list ap;
   va_start (ap,msg);
   vprintf (msg,ap);
+  va_end (ap);
 }
 
 static void captureFrame( int num ){
--- drawstuff/src/windows.cpp	Mon Oct 15 13:18:55 2012
+++ drawstuff/src/windows.cpp	Mon Oct 15 13:19:15 2012
@@ -54,6 +54,7 @@
   va_list ap;
   va_start (ap,msg);
   errorBox ("Warning",msg,ap);
+  va_end (ap);
 }
 
 
@@ -62,6 +63,7 @@
   va_list ap;
   va_start (ap,msg);
   errorBox ("Error",msg,ap);
+  va_end (ap);
   exit (1);
 }
 
@@ -72,6 +74,7 @@
   va_start (ap,msg);
   errorBox ("INTERNAL ERROR",msg,ap);
   // *((char *)0) = 0;	 ... commit SEGVicide ?
+  va_end (ap);
   abort();
   exit (1);	  // should never get here, but just in case...
 }
@@ -82,6 +85,7 @@
   va_list ap;
   va_start (ap,msg);
   vprintf (msg,ap);
+  va_end (ap);
 }
 
 //***************************************************************************
--- drawstuff/src/x11.cpp	Mon Oct 15 13:19:50 2012
+++ drawstuff/src/x11.cpp	Mon Oct 15 13:19:58 2012
@@ -62,6 +62,7 @@
   va_list ap;
   va_start (ap,msg);
   printMessage ("Error",msg,ap);
+  va_end (ap);
   exit (1);
 }
 
@@ -72,6 +73,7 @@
   va_start (ap,msg);
   printMessage ("INTERNAL ERROR",msg,ap);
   // *((char *)0) = 0;	 ... commit SEGVicide ?
+  va_end (ap);
   abort();
 }
 
@@ -81,6 +83,7 @@
   va_list ap;
   va_start (ap,msg);
   vprintf (msg,ap);
+  va_end (ap);
 }
 
 //***************************************************************************
