--- OpenCV-2.3.0/CMakeLists.txt	2011-09-23 14:01:43 +0000
+++ OpenCV-2.3.0_patched/CMakeLists.txt	2011-09-23 14:03:55 +0000
@@ -1333,9 +1333,7 @@
 
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/opencv.pc.cmake.in" "${CMAKE_BINARY_DIR}/unix-install/opencv.pc" @ONLY IMMEDIATE)
 
-if(UNIX)
-    install(FILES ${CMAKE_BINARY_DIR}/unix-install/opencv.pc DESTINATION lib/pkgconfig)
-endif()
+install(FILES ${CMAKE_BINARY_DIR}/unix-install/opencv.pc DESTINATION lib/pkgconfig)
 
 
 # ----------------------------------------------------------------------------
