diff -rupN torcs-1.3.4_clean/configure torcs-1.3.4/configure
--- torcs-1.3.4_clean/configure	2013-01-09 14:10:23.000000000 +0100
+++ torcs-1.3.4/configure	2013-01-09 15:38:51.000000000 +0100
@@ -14,6 +14,10 @@
 ## M4sh Initialization. ##
 ## -------------------- ##
 
+## for lion
+CPPFLAGS="-I/opt/local/include -I$(PWD) -I/Users/malter/Arbeit/mars_dev/install/include";
+LDFLAGS="-L/opt/local/lib -L$(PWD) -undefined dynamic_lookup";
+
 # Be more Bourne compatible
 DUALCASE=1; export DUALCASE # for MKS sh
 if test -n "${ZSH_VERSION+set}" && (emulate sh) >/dev/null 2>&1; then :
@@ -5432,24 +5436,24 @@ CXXFLAGS="$CXXFLAGS $ADDCFLAGS"
 ac_fn_cxx_check_header_mongrel "$LINENO" "GL/gl.h" "ac_cv_header_GL_gl_h" "$ac_includes_default"
 if test "x$ac_cv_header_GL_gl_h" = xyes; then :
 
-else
-  as_fn_error $? "Can't find GL/gl.h. Look for Mesa devel packages for your distro." "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find GL/gl.h. Look for Mesa devel packages for your distro." "$LINENO" 5
 fi
 
 
 ac_fn_cxx_check_header_mongrel "$LINENO" "GL/glut.h" "ac_cv_header_GL_glut_h" "$ac_includes_default"
 if test "x$ac_cv_header_GL_glut_h" = xyes; then :
 
-else
-  as_fn_error $? "Can't find GL/glut.h. freeglut can be found on http://freeglut.sourceforge.net/" "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find GL/glut.h. freeglut can be found on http://freeglut.sourceforge.net/" "$LINENO" 5
 fi
 
 
 ac_fn_cxx_check_header_mongrel "$LINENO" "GL/glx.h" "ac_cv_header_GL_glx_h" "$ac_includes_default"
 if test "x$ac_cv_header_GL_glx_h" = xyes; then :
 
-else
-  as_fn_error $? "Can't find GL/glx.h. Please check config.log and if you can't solve the problem send the file to torcs-users@lists.sourceforge.net with the subject \"torcs compilation problem\"" "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find GL/glx.h. Please check config.log and if you can't solve the problem send the file to torcs-users@lists.sourceforge.net with the subject \"torcs compilation problem\"" "$LINENO" 5
 fi
 
 
@@ -5512,8 +5516,8 @@ fi
 ac_fn_cxx_check_header_mongrel "$LINENO" "AL/al.h" "ac_cv_header_AL_al_h" "$ac_includes_default"
 if test "x$ac_cv_header_AL_al_h" = xyes; then :
 
-else
-  as_fn_error $? "Can't find AL/al.h. OpenAL and FreeALUT can be found on http://www.openal.org/" "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find AL/al.h. OpenAL and FreeALUT can be found on http://www.openal.org/" "$LINENO" 5
 fi
 
 
@@ -6099,8 +6103,8 @@ fi
 $as_echo "$ac_cv_lib_openal_alEnable" >&6; }
 if test "x$ac_cv_lib_openal_alEnable" = xyes; then :
   LDFLAGS="$LDFLAGS -lopenal"
-else
-  as_fn_error $? "Can't find AL/al.h. OpenAL can be found on http://www.openal.org/" "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find AL/al.h. OpenAL can be found on http://www.openal.org/" "$LINENO" 5
 fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for alutLoadWAVFile in -lalut" >&5
@@ -6609,8 +6613,8 @@ _ACEOF
 
   LIBS="-lplibsl $LIBS"
 
-else
-  as_fn_error $? "Can't find libplibsl. PLIB can be found on http://plib.sourceforge.net/" "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find libplibsl. PLIB can be found on http://plib.sourceforge.net/" "$LINENO" 5
 fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lplibsm" >&5
@@ -6650,8 +6654,8 @@ _ACEOF
 
   LIBS="-lplibsm $LIBS"
 
-else
-  as_fn_error $? "Can't find libplibsm. PLIB can be found on http://plib.sourceforge.net/" "$LINENO" 5
+#else
+  #as_fn_error $? "Can't find libplibsm. PLIB can be found on http://plib.sourceforge.net/" "$LINENO" 5
 fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lplibssg" >&5
diff -rupN torcs-1.3.4_clean/src/drivers/olethros/driver.cpp torcs-1.3.4/src/drivers/olethros/driver.cpp
--- torcs-1.3.4_clean/src/drivers/olethros/driver.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/drivers/olethros/driver.cpp	2013-01-09 18:09:14.000000000 +0100
@@ -43,6 +43,10 @@
 #define isnan _isnan
 #endif
 
+#ifdef __APPLE__
+#define isnan std::isnan
+#endif
+
 #undef TEST_PITSTOPS
 
 #ifdef USE_OLETHROS_NAMESPACE
diff -rupN torcs-1.3.4_clean/src/drivers/olethros/geometry.cpp torcs-1.3.4/src/drivers/olethros/geometry.cpp
--- torcs-1.3.4_clean/src/drivers/olethros/geometry.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/drivers/olethros/geometry.cpp	2013-01-09 18:10:14.000000000 +0100
@@ -29,6 +29,10 @@
 #define isnan _isnan
 #endif
 
+#ifdef __APPLE__
+#define isnan std::isnan
+#endif
+
 
 Vector::Vector(int N_, enum BoundsCheckingStatus check)
 {
diff -rupN torcs-1.3.4_clean/src/libs/Makefile torcs-1.3.4/src/libs/Makefile
--- torcs-1.3.4_clean/src/libs/Makefile	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/Makefile	2013-01-10 17:36:38.000000000 +0100
@@ -19,12 +19,12 @@
 SHIPSUBDIRS	= tgf tgfclient raceengineclient learning musicplayer
 
 EXPINCDIRS	= txml tgf tgfclient client confscreens racescreens robottools raceengineclient learning \
-			  portability math musicplayer
+			  portability math #musicplayer
 
 TOOLSUBDIRS	= txml
 
 SUBDIRS		= confscreens racescreens robottools txml tgf tgfclient client raceengineclient learning \
-			  portability math musicplayer
+			  portability math #musicplayer
 
 PKGSUBDIRS	= $(SUBDIRS)
 
diff -rupN torcs-1.3.4_clean/src/libs/client/entry.cpp torcs-1.3.4/src/libs/client/entry.cpp
--- torcs-1.3.4_clean/src/libs/client/entry.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/client/entry.cpp	2013-01-10 17:43:14.000000000 +0100
@@ -24,7 +24,7 @@
 
 #include "mainmenu.h"
 #include "splash.h"
-#include <musicplayer/musicplayer.h>
+//#include <musicplayer/musicplayer.h>
 
 /*
  * Function
@@ -53,5 +53,5 @@ TorcsEntry(void)
 
 	SplashScreen();
 	
-	startMenuMusic();
+	//startMenuMusic();
 }
diff -rupN torcs-1.3.4_clean/src/libs/confscreens/soundconfig.cpp torcs-1.3.4/src/libs/confscreens/soundconfig.cpp
--- torcs-1.3.4_clean/src/libs/confscreens/soundconfig.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/confscreens/soundconfig.cpp	2013-01-10 17:46:05.000000000 +0100
@@ -133,9 +133,9 @@ static void saveSoundOption(void *)
 	GfParmReleaseHandle(paramHandle);
 	
 	if (curOptionMenuMusic == 1) {
-		startMenuMusic();
+          //startMenuMusic();
 	} else {
-		stopMenuMusic();
+          //stopMenuMusic();
 	}
 	
 	// Return to previous screen.
diff -rupN torcs-1.3.4_clean/src/libs/learning/policy.cpp torcs-1.3.4/src/libs/learning/policy.cpp
--- torcs-1.3.4_clean/src/libs/learning/policy.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/learning/policy.cpp	2013-01-09 15:41:06.000000000 +0100
@@ -18,6 +18,8 @@
 #ifdef WIN32
 #include <float.h>
 #define isnan _isnan
+#elif defined(__APPLE__)
+#define isnan std::isnan
 #endif // WIN32
 
 #undef POLICY_LOG
diff -rupN torcs-1.3.4_clean/src/libs/musicplayer/OpenALMusicPlayer.h torcs-1.3.4/src/libs/musicplayer/OpenALMusicPlayer.h
--- torcs-1.3.4_clean/src/libs/musicplayer/OpenALMusicPlayer.h	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/musicplayer/OpenALMusicPlayer.h	2013-01-09 15:44:59.000000000 +0100
@@ -20,8 +20,13 @@
  *                                                                         *
  ***************************************************************************/
 
+#ifdef __APPLE__
+#include <OpenAL/al.h>
+#include <OpenAL/alc.h>
+#else
 #include <AL/al.h>
 #include <AL/alc.h>
+#endif
 #include "SoundStream.h"
 
 class OpenALMusicPlayer
@@ -53,4 +58,4 @@ class OpenALMusicPlayer
 		bool ready;									// initialization sucessful
 		static const int BUFFERSIZE;
 };
-#endif // __OpenALMusicPlayer_h__
\ No newline at end of file
+#endif // __OpenALMusicPlayer_h__
Binary files torcs-1.3.4_clean/src/libs/musicplayer/OpenALMusicPlayer.o and torcs-1.3.4/src/libs/musicplayer/OpenALMusicPlayer.o differ
Binary files torcs-1.3.4_clean/src/libs/musicplayer/SoundStream.o and torcs-1.3.4/src/libs/musicplayer/SoundStream.o differ
Binary files torcs-1.3.4_clean/src/libs/musicplayer/libmusicplayer.so and torcs-1.3.4/src/libs/musicplayer/libmusicplayer.so differ
Binary files torcs-1.3.4_clean/src/libs/musicplayer/musicplayer.o and torcs-1.3.4/src/libs/musicplayer/musicplayer.o differ
diff -rupN torcs-1.3.4_clean/src/libs/raceengineclient/raceinit.cpp torcs-1.3.4/src/libs/raceengineclient/raceinit.cpp
--- torcs-1.3.4_clean/src/libs/raceengineclient/raceinit.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/raceengineclient/raceinit.cpp	2013-01-10 17:44:41.000000000 +0100
@@ -37,7 +37,7 @@
 #include <string>
 #include <map>
 #include <portability.h>
-#include <musicplayer/musicplayer.h>
+//#include <musicplayer/musicplayer.h>
 
 #include "raceengine.h"
 #include "racemain.h"
@@ -823,7 +823,7 @@ ReRaceCleanup(void)
 	ReInfo->_reSimItf.shutdown();
 	if (ReInfo->_displayMode == RM_DISP_MODE_NORMAL) {
 		ReInfo->_reGraphicItf.shutdowncars();
-		startMenuMusic();
+		//startMenuMusic();
 	}
 	ReStoreRaceResults(ReInfo->_reRaceName);
 	ReRaceCleanDrivers();
diff -rupN torcs-1.3.4_clean/src/libs/raceengineclient/racemain.cpp torcs-1.3.4/src/libs/raceengineclient/racemain.cpp
--- torcs-1.3.4_clean/src/libs/raceengineclient/racemain.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/raceengineclient/racemain.cpp	2013-01-10 17:45:01.000000000 +0100
@@ -30,7 +30,7 @@
 #include <robot.h>
 #include <racescreens.h>
 #include <exitmenu.h>
-#include <musicplayer/musicplayer.h>
+//#include <musicplayer/musicplayer.h>
 #include <portability.h>
 
 #include "raceengine.h"
@@ -81,7 +81,7 @@ AbortRaceHookActivate(void * /* dummy */
 	ReInfo->_reSimItf.shutdown();
 	if (ReInfo->_displayMode == RM_DISP_MODE_NORMAL) {
 		ReInfo->_reGraphicItf.shutdowncars();
-		startMenuMusic();
+		//startMenuMusic();
 	}
 	ReInfo->_reGraphicItf.shutdowntrack();
 	ReRaceCleanDrivers();
@@ -267,7 +267,7 @@ reRaceRealStart(void)
 
 		if (ReInfo->_displayMode == RM_DISP_MODE_NORMAL) {
 			/* RmLoadingScreenSetText("Loading Cars 3D Objects..."); */
-			stopMenuMusic();
+			//stopMenuMusic();
 			ReInfo->_reGraphicItf.initcars(s);
 		}
 
diff -rupN torcs-1.3.4_clean/src/libs/tgfclient/guifont.cpp torcs-1.3.4/src/libs/tgfclient/guifont.cpp
--- torcs-1.3.4_clean/src/libs/tgfclient/guifont.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/tgfclient/guifont.cpp	2013-01-09 15:40:06.000000000 +0100
@@ -24,7 +24,7 @@
 
 #ifdef WIN32
 #include <windows.h>
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) || defined(__APPLE__)
 #include <machine/endian.h>
 #elif defined(sun)
 #define BIG_ENDIAN 1234
diff -rupN torcs-1.3.4_clean/src/libs/tgfclient/screen.cpp torcs-1.3.4/src/libs/tgfclient/screen.cpp
--- torcs-1.3.4_clean/src/libs/tgfclient/screen.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/libs/tgfclient/screen.cpp	2013-01-10 17:45:24.000000000 +0100
@@ -37,7 +37,7 @@
 
 #include <tgfclient.h>
 #include <portability.h>
-#include <musicplayer/musicplayer.h>
+//#include <musicplayer/musicplayer.h>
 #include <portability.h>
 
 #include "gui.h"
@@ -493,7 +493,7 @@ GfScrReinit(void * /* dummy */)
 	static const int CMDSIZE = 1024;
 	char cmd[CMDSIZE];
 
-	stopMenuMusic();
+	//stopMenuMusic();
 	
 #ifndef WIN32
     const char *arg[8];
diff -rupN torcs-1.3.4_clean/src/linux/Makefile torcs-1.3.4/src/linux/Makefile
--- torcs-1.3.4_clean/src/linux/Makefile	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/linux/Makefile	2013-01-10 17:36:56.000000000 +0100
@@ -34,8 +34,8 @@ SOLIBS  =  -lracescreens  \
 	   -ltxml	  \
 	   -lplibul	  \
 	   -lraceengine   \
-	   -lmusicplayer  \
 	   -llearning
+#	   -lmusicplayer  \
 
 EXPDIR  = include
 
diff -rupN torcs-1.3.4_clean/src/modules/graphic/ssggraph/Makefile torcs-1.3.4/src/modules/graphic/ssggraph/Makefile
--- torcs-1.3.4_clean/src/modules/graphic/ssggraph/Makefile	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/modules/graphic/ssggraph/Makefile	2013-01-09 15:50:43.000000000 +0100
@@ -18,13 +18,13 @@
 
 SOURCES   = grmain.cpp grboard.cpp ssggraph.cpp grcam.cpp grcar.cpp grscreen.cpp\
 	grscene.cpp grutil.cpp grshadow.cpp grsmoke.cpp grskidmarks.cpp grloadac.cpp\
-	grmultitexstate.cpp grvtxtable.cpp grtrackmap.cpp grtexture.cpp grcarlight.cpp CarSoundData.cpp TorcsSound.cpp PlibSoundInterface.cpp OpenalSoundInterface.cpp grsound.cpp SoundInterface.cpp
+	grmultitexstate.cpp grvtxtable.cpp grtrackmap.cpp grtexture.cpp grcarlight.cpp #CarSoundData.cpp TorcsSound.cpp PlibSoundInterface.cpp OpenalSoundInterface.cpp grsound.cpp SoundInterface.cpp
 
 MODULE    = ssggraph.so
 
 MODULEDIR = modules/graphic
 
-LIBS      = -lplibsl -lplibsm -lplibsg -lplibssg -lplibul -lplibssgaux -lopenal
+LIBS      = -lplibsg -lplibssg -lplibul -lplibssgaux #-lopenal-lplibsl -lplibsm 
 
 SHIPDIR   = config
 
diff -rupN torcs-1.3.4_clean/src/modules/graphic/ssggraph/grmain.cpp torcs-1.3.4/src/modules/graphic/ssggraph/grmain.cpp
--- torcs-1.3.4_clean/src/modules/graphic/ssggraph/grmain.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/modules/graphic/ssggraph/grmain.cpp	2013-01-09 15:52:39.000000000 +0100
@@ -38,7 +38,7 @@
 #include "grscreen.h"
 #include "grcam.h"
 #include "grscene.h"
-#include "grsound.h"
+//#include "grsound.h"
 #include "grboard.h"
 #include "grutil.h"
 #include "grtrackmap.h"
@@ -324,9 +324,9 @@ refresh(tSituation *s)
 
     TRACE_GL("refresh: start");
 
-    START_PROFILE("grRefreshSound*");
-    grRefreshSound(s, grScreens[0]->getCurCamera());
-    STOP_PROFILE("grRefreshSound*");
+    //START_PROFILE("grRefreshSound*");
+    //grRefreshSound(s, grScreens[0]->getCurCamera());
+    //STOP_PROFILE("grRefreshSound*");
 
     START_PROFILE("grDrawBackground/glClear");
     glDepthFunc(GL_LEQUAL);
@@ -405,7 +405,7 @@ initCars(tSituation *s)
 	TRACE_GL("initCars: end");
 
 	grInitSmoke(s->_ncars);
-	grInitSound(s, s->_ncars);
+	//grInitSound(s, s->_ncars);
 
 	grAdaptScreenSize();
 
@@ -418,7 +418,7 @@ shutdownCars(void)
 	int i;
 
 	GfOut("-- shutdownCars\n");
-	grShutdownSound(grNbCars);
+	//grShutdownSound(grNbCars);
 	if (grNbCars) {
 		grShutdownBoardCar();
 		grShutdownSkidmarks();
diff -rupN torcs-1.3.4_clean/src/modules/simu/simuv2/SOLID-2.0/src/C-api.cpp torcs-1.3.4/src/modules/simu/simuv2/SOLID-2.0/src/C-api.cpp
--- torcs-1.3.4_clean/src/modules/simu/simuv2/SOLID-2.0/src/C-api.cpp	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/modules/simu/simuv2/SOLID-2.0/src/C-api.cpp	2013-01-09 15:46:13.000000000 +0100
@@ -44,7 +44,7 @@
 #include "Polyhedron.h"
 #include "Response.h"
 #include "RespTable.h"
-#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__)
+#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__APPLE__)
 #  include <sys/types.h>
 #endif
 
diff -rupN torcs-1.3.4_clean/src/tools/texmapper/Makefile torcs-1.3.4/src/tools/texmapper/Makefile
--- torcs-1.3.4_clean/src/tools/texmapper/Makefile	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/tools/texmapper/Makefile	2013-01-10 17:37:35.000000000 +0100
@@ -24,7 +24,7 @@ PROGRAM   	= texmapper-bin
 
 SOURCES   	= maintexmapper.cpp ssgSaveAC.cpp
 
-SOLIBS    	= -lplibssg -lplibsg -lplibul -ltgf -ltgfclient -lmusicplayer -ltxml -lpng
+SOLIBS    	= -lplibssg -lplibsg -lplibul -ltgf -ltgfclient -ltxml -lpng # -lmusicplayer
 
 src_PKGFILES	= $(shell find * -maxdepth 0 -type f -print)
 
diff -rupN torcs-1.3.4_clean/src/tools/trackgen/Makefile torcs-1.3.4/src/tools/trackgen/Makefile
--- torcs-1.3.4_clean/src/tools/trackgen/Makefile	2013-01-09 14:10:36.000000000 +0100
+++ torcs-1.3.4/src/tools/trackgen/Makefile	2013-01-10 17:37:53.000000000 +0100
@@ -29,7 +29,7 @@ DATA         	= trackgen.xml
 
 SOURCES      	= maintrackgen.cpp track.cpp easymesh.cpp ../../linux/linuxspec.cpp relief.cpp elevation.cpp objects.cpp ac3d.cpp util.cpp raceline.cpp
 
-SOLIBS       	= -ltgf -ltgfclient -lmusicplayer -lplibssg -lplibsg -lplibul -lrobottools -ltxml
+SOLIBS       	= -ltgf -ltgfclient -lplibssg -lplibsg -lplibul -lrobottools -ltxml # -lmusicplayer
 
 src_PKGFILES	= $(shell find * -maxdepth 0 -type f -print)
 
