--- yaml-cpp/CMakeLists.txt	2012-10-25 09:17:45 +0000
+++ yaml-cpp/CMakeLists.txt	2012-10-25 09:18:11 +0000
@@ -205,7 +205,7 @@
 ###
 ### General install settings
 ###
-if(WIN32)
+if(WIN32 AND BUILD_SHARED_LIBS)
 	set(_library_dir bin)	# .dll are in PATH, like executables
 else()
 	set(_library_dir lib)
@@ -266,11 +266,18 @@
 	FILES_MATCHING PATTERN "*.h"
 )
 
-if(UNIX)
-	set(PC_FILE ${CMAKE_BINARY_DIR}/yaml-cpp.pc)
-	configure_file("yaml-cpp.pc.cmake" ${PC_FILE} @ONLY)
-	install(FILES ${PC_FILE} DESTINATION ${LIB_INSTALL_DIR}/pkgconfig)
+set(PC_FILE ${CMAKE_BINARY_DIR}/yaml-cpp.pc)
+configure_file("yaml-cpp.pc.cmake" ${PC_FILE} @ONLY)
+if(WIN32)
+	set(PKG_CONFIG_DIR lib${LIB_SUFFIX}/pkgconfig)
+       # this fixes the error 998 from the LibManager
+       set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--enable-auto-import")
+       set(CMAKE_MODULE_LINKER_FLAGS "-Wl,--enable-auto-import")
+else()
+	set(PKG_CONFIG_DIR ${LIB_INSTALL_DIR}/pkgconfig)
+       set(CMAKE_CXX_FLAGS "-fPIC")
 endif()
+install(FILES ${PC_FILE} DESTINATION ${PKG_CONFIG_DIR})
 
 
 ###
